packages/vm/package.json

```diff
@@ -46,21 +46,21 @@
     "VM"
   ],
   "dependencies": {
-    "async": "^2.1.2",
-    "async-eventemitter": "^0.2.2",
+    "async": "^3.2.0",
+    "async-eventemitter": "^0.2.4",
     "core-js-pure": "^3.0.1",
     "ethereumjs-account": "^3.0.0",
     "ethereumjs-block": "^2.2.2",
     "ethereumjs-blockchain": "^4.0.3",
     "ethereumjs-common": "^1.5.0",
     "ethereumjs-tx": "^2.1.2",
     "ethereumjs-util": "^6.2.0",
-    "fake-merkle-patricia-tree": "^1.0.1",
     "functional-red-black-tree": "^1.0.1",
     "merkle-patricia-tree": "^2.3.2",
     "rustbn.js": "~0.2.0",
     "safe-buffer": "^5.1.1",
-    "util.promisify": "^1.0.0"
+    "util-promisify": "^2.1.0",
```

Comment:
Why is this switch from [util.promisify](https://github.com/ljharb/util.promisify) to [util-promisify](https://github.com/juliangruber/util-promisify)?

Reply:
I originally wanted to remove this dep altogether since it's included since node 8 and browserify should shim it properly, but I couldn't get it working in karma with our setup - I think due to some complexity with our imports. I must have typo'd putting it back because the other import is called `util-callbackify` 😄 so I'll fix, looks like the one with the period is better maintained.